Description: debianization: nauty tools suite prefix
 Prepend nauty- to the names of the nauty tools, named gtools.
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-03-19

--- a/Makefile.am
+++ b/Makefile.am
@@ -21,94 +21,94 @@
 bin_PROGRAMS = \
 	dreadnaut \
 	\
-	addedgeg \
-	amtog \
-	biplabg \
-	catg \
-	complg \
-	converseg \
-	copyg \
-	countg \
-	cubhamg \
-	deledgeg \
-	delptg \
-	directg \
-	dretodot \
-	dretog \
-	genbg \
-	genbgL \
-	geng \
-	genquarticg \
-	genrang \
-	genspecialg \
-	gentourng \
-	gentreeg \
-	hamheuristic \
-	labelg \
-	linegraphg \
-	listg \
-	multig \
-	newedgeg \
-	NRswitchg \
-	pickg \
-	planarg \
-	ranlabg \
-	shortg \
-	showg \
-	subdivideg \
-	twohamg \
-	vcolg \
-	watercluster2 \
+	nauty-addedgeg \
+	nauty-amtog \
+	nauty-biplabg \
+	nauty-catg \
+	nauty-complg \
+	nauty-converseg \
+	nauty-copyg \
+	nauty-countg \
+	nauty-cubhamg \
+	nauty-deledgeg \
+	nauty-delptg \
+	nauty-directg \
+	nauty-dretodot \
+	nauty-dretog \
+	nauty-genbg \
+	nauty-genbgL \
+	nauty-geng \
+	nauty-genquarticg \
+	nauty-genrang \
+	nauty-genspecialg \
+	nauty-gentourng \
+	nauty-gentreeg \
+	nauty-hamheuristic \
+	nauty-labelg \
+	nauty-linegraphg \
+	nauty-listg \
+	nauty-multig \
+	nauty-newedgeg \
+	nauty-NRswitchg \
+	nauty-pickg \
+	nauty-planarg \
+	nauty-ranlabg \
+	nauty-shortg \
+	nauty-showg \
+	nauty-subdivideg \
+	nauty-twohamg \
+	nauty-vcolg \
+	nauty-watercluster2 \
 	\
-	blisstog \
-	checks6 \
-	sumlines
+	nauty-blisstog \
+	nauty-checks6 \
+	nauty-sumlines
 
 man_MANS = \
 	dreadnaut.1 \
 	\
-	addedgeg.1 \
-	amtog.1 \
-	biplabg.1 \
-	catg.1 \
-	complg.1 \
-	converseg.1 \
-	copyg.1 \
-	countg.1 \
-	cubhamg.1 \
-	deledgeg.1 \
-	delptg.1 \
-	directg.1 \
-	dretodot.1 \
-	dretog.1 \
-	genbg.1 \
-	genbgL.1 \
-	geng.1 \
-	genquarticg.1 \
-	genrang.1 \
-	genspecialg.1 \
-	gentourng.1 \
-	gentreeg.1 \
-	hamheuristic.1 \
-	labelg.1 \
-	linegraphg.1 \
-	listg.1 \
-	multig.1 \
-	newedgeg.1 \
-	NRswitchg.1 \
-	pickg.1 \
-	planarg.1 \
-	ranlabg.1 \
-	shortg.1 \
-	showg.1 \
-	subdivideg.1 \
-	twohamg.1 \
-	vcolg.1 \
-	watercluster2.1 \
+	nauty-addedgeg.1 \
+	nauty-amtog.1 \
+	nauty-biplabg.1 \
+	nauty-catg.1 \
+	nauty-complg.1 \
+	nauty-converseg.1 \
+	nauty-copyg.1 \
+	nauty-countg.1 \
+	nauty-cubhamg.1 \
+	nauty-deledgeg.1 \
+	nauty-delptg.1 \
+	nauty-directg.1 \
+	nauty-dretodot.1 \
+	nauty-dretog.1 \
+	nauty-genbg.1 \
+	nauty-genbgL.1 \
+	nauty-geng.1 \
+	nauty-genquarticg.1 \
+	nauty-genrang.1 \
+	nauty-genspecialg.1 \
+	nauty-gentourng.1 \
+	nauty-gentreeg.1 \
+	nauty-hamheuristic.1 \
+	nauty-labelg.1 \
+	nauty-linegraphg.1 \
+	nauty-listg.1 \
+	nauty-multig.1 \
+	nauty-newedgeg.1 \
+	nauty-NRswitchg.1 \
+	nauty-pickg.1 \
+	nauty-planarg.1 \
+	nauty-ranlabg.1 \
+	nauty-shortg.1 \
+	nauty-showg.1 \
+	nauty-subdivideg.1 \
+	nauty-twohamg.1 \
+	nauty-vcolg.1 \
+	nauty-watercluster2.1 \
 	\
-	blisstog.1 \
-	checks6.1 \
-	sumlines.1
+	nauty-blisstog.1 \
+	nauty-checks6.1 \
+	nauty-sumlines.1
 
 lib_LTLIBRARIES = \
 	libnauty.la \
@@ -153,8 +153,8 @@
 	$(HELP2MAN) \
 			-s 1 \
 			$(AM_H2MFLAGS) \
-			-I $(top_srcdir)/man/$*.h2m \
-			-n $(if $($*_DESCRIPTION), $($*_DESCRIPTION), "manual page for $*") \
+			-I $(top_srcdir)/man/$(subst nauty-,,$*).h2m \
+			-n $(if $($(subst -,_,$*)_DESCRIPTION), $($(subst -,_,$*)_DESCRIPTION), "manual page for $*") \
 			-o $@ \
 		$(top_builddir)/$<
 
@@ -175,176 +175,176 @@
 ## short descriptions inspired from the basic functions of the programs as described
 ## in section `Utilities' (15) of the `Nauty and Traces User's Guide' (version 2.6)
 dreadnaut_DESCRIPTION =     "command line interface to nauty graph isomorphism library"
-addedgeg_DESCRIPTION =      "add an edge in each possible way"
-amtog_DESCRIPTION =         "read graphs in matrix format"
-biplabg_DESCRIPTION =       "label bipartite graphs so the colour classes are contiguous"
-catg_DESCRIPTION =          "concatenate files of graphs"
-complg_DESCRIPTION =        "complement graphs"
-converseg_DESCRIPTION =     "converse digraphs"
-copyg_DESCRIPTION =         "convert format and select subset"
-countg_DESCRIPTION =        "count graphs according to a variety of properties"
-cubhamg_DESCRIPTION =       "find hamiltonian cycles in subcubic graphs"
-deledgeg_DESCRIPTION =      "delete an edge in each possible way"
-delptg_DESCRIPTION =        "delete vertices"
-directg_DESCRIPTION =       "generate small digraphs with given underlying graph"
-dretodot_DESCRIPTION =      "read graphs and initial coloring in dreadnaut format and write in dot format"
-dretog_DESCRIPTION =        "read graphs in dreadnaut format"
-genbg_DESCRIPTION =         "generate small bicoloured graphs"
-genbgL_DESCRIPTION =        "$(shell printf "%s %s" $(genbg_DESCRIPTION) "(L1 flavour)")"
-geng_DESCRIPTION =          "generate small graphs"
-genquarticg_DESCRIPTION =   "generate quartic graphs"
-genrang_DESCRIPTION =       "generate random graphs"
-genspecialg_DESCRIPTION =   "generate special graphs"
-gentourng_DESCRIPTION =     "generate small tournaments"
-gentreeg_DESCRIPTION =      "generate trees"
-hamheuristic_DESCRIPTION =  "try to find hamiltonian cycles in any graphs"
-labelg_DESCRIPTION =        "canonically label graphs"
-linegraphg_DESCRIPTION =    "compute the linegraphs of a file of graphs"
-listg_DESCRIPTION =         "display graphs in a variety of forms"
-multig_DESCRIPTION =        "generate small multigraphs with given underlying graph"
-newedgeg_DESCRIPTION =      "create new edges from pairs of non-adjacent edges in each possible way"
-NRswitchg_DESCRIPTION =     "complement vertex edges"
-pickg_DESCRIPTION =         "select graphs according to a variety of properties"
-planarg_DESCRIPTION =       "test graphs for planarity and find embeddings or obstructions"
-ranlabg_DESCRIPTION =       "randomly relabel graphs"
-shortg_DESCRIPTION =        "remove isomorphs from a file of graphs"
-showg_DESCRIPTION =         "$(shell printf "%s %s" $(listg_DESCRIPTION) "(stand-alone subset of listg)")"
-subdivideg_DESCRIPTION =    "compute the subdivision graphs of a file of graphs"
-twohamg_DESCRIPTION =       "split quartic graphs into two hamiltonian cycles"
-vcolg_DESCRIPTION =         "colour the vertices of graphs in all distinct ways"
-watercluster2_DESCRIPTION = "$(shell printf "%s %s" $(directg_DESCRIPTION) "(faster alternative to directg)")"
-blisstog_DESCRIPTION =      "convert files of graphs in bliss format to stdout in sparse6 format"
-checks6_DESCRIPTION =       "check a file of graphs and optionally perform corrections"
-sumlines_DESCRIPTION =      "sum lines matching specified graph formats"
+nauty_addedgeg_DESCRIPTION =      "add an edge in each possible way"
+nauty_amtog_DESCRIPTION =         "read graphs in matrix format"
+nauty_biplabg_DESCRIPTION =       "label bipartite graphs so the colour classes are contiguous"
+nauty_catg_DESCRIPTION =          "concatenate files of graphs"
+nauty_complg_DESCRIPTION =        "complement graphs"
+nauty_converseg_DESCRIPTION =     "converse digraphs"
+nauty_copyg_DESCRIPTION =         "convert format and select subset"
+nauty_countg_DESCRIPTION =        "count graphs according to a variety of properties"
+nauty_cubhamg_DESCRIPTION =       "find hamiltonian cycles in subcubic graphs"
+nauty_deledgeg_DESCRIPTION =      "delete an edge in each possible way"
+nauty_delptg_DESCRIPTION =        "delete vertices"
+nauty_directg_DESCRIPTION =       "generate small digraphs with given underlying graph"
+nauty_dretodot_DESCRIPTION =      "read graphs and initial coloring in dreadnaut format and write in dot format"
+nauty_dretog_DESCRIPTION =        "read graphs in dreadnaut format"
+nauty_genbg_DESCRIPTION =         "generate small bicoloured graphs"
+nauty_genbgL_DESCRIPTION =        "$(shell printf "%s %s" $(nauty_genbg_DESCRIPTION) "(L1 flavour)")"
+nauty_geng_DESCRIPTION =          "generate small graphs"
+nauty_genquarticg_DESCRIPTION =   "generate quartic graphs"
+nauty_genrang_DESCRIPTION =       "generate random graphs"
+nauty_genspecialg_DESCRIPTION =   "generate special graphs"
+nauty_gentourng_DESCRIPTION =     "generate small tournaments"
+nauty_gentreeg_DESCRIPTION =      "generate trees"
+nauty_hamheuristic_DESCRIPTION =  "try to find hamiltonian cycles in any graphs"
+nauty_labelg_DESCRIPTION =        "canonically label graphs"
+nauty_linegraphg_DESCRIPTION =    "compute the linegraphs of a file of graphs"
+nauty_listg_DESCRIPTION =         "display graphs in a variety of forms"
+nauty_multig_DESCRIPTION =        "generate small multigraphs with given underlying graph"
+nauty_newedgeg_DESCRIPTION =      "create new edges from pairs of non-adjacent edges in each possible way"
+nauty_NRswitchg_DESCRIPTION =     "complement vertex edges"
+nauty_pickg_DESCRIPTION =         "select graphs according to a variety of properties"
+nauty_planarg_DESCRIPTION =       "test graphs for planarity and find embeddings or obstructions"
+nauty_ranlabg_DESCRIPTION =       "randomly relabel graphs"
+nauty_shortg_DESCRIPTION =        "remove isomorphs from a file of graphs"
+nauty_showg_DESCRIPTION =         "$(shell printf "%s %s" $(nauty_listg_DESCRIPTION) "(stand-alone subset of listg)")"
+nauty_subdivideg_DESCRIPTION =    "compute the subdivision graphs of a file of graphs"
+nauty_twohamg_DESCRIPTION =       "split quartic graphs into two hamiltonian cycles"
+nauty_vcolg_DESCRIPTION =         "colour the vertices of graphs in all distinct ways"
+nauty_watercluster2_DESCRIPTION = "$(shell printf "%s %s" $(nauty_directg_DESCRIPTION) "(faster alternative to directg)")"
+nauty_blisstog_DESCRIPTION =      "convert files of graphs in bliss format to stdout in sparse6 format"
+nauty_checks6_DESCRIPTION =       "check a file of graphs and optionally perform corrections"
+nauty_sumlines_DESCRIPTION =      "sum lines matching specified graph formats"
 
 dreadnaut_CPPFLAGS =
-addedgeg_CPPFLAGS =
-amtog_CPPFLAGS =
-biplabg_CPPFLAGS =
-catg_CPPFLAGS =
-complg_CPPFLAGS =
-converseg_CPPFLAGS =
-copyg_CPPFLAGS =
-countg_CPPFLAGS =
-cubhamg_CPPFLAGS =
-deledgeg_CPPFLAGS =
-delptg_CPPFLAGS =
-directg_CPPFLAGS =
-dretodot_CPPFLAGS =
-dretog_CPPFLAGS =
-genbg_CPPFLAGS =         $(nautyW1_flavour_CPPFLAGS)
-genbgL_CPPFLAGS =        $(nautyL1_flavour_CPPFLAGS) -DMAXN1=30
-geng_CPPFLAGS =          $(nautyW1_flavour_CPPFLAGS)
-genquarticg_CPPFLAGS =   $(nautyL1_flavour_CPPFLAGS)
-genrang_CPPFLAGS =
-genspecialg_CPPFLAGS =
-gentourng_CPPFLAGS =     -DMAXN=24
-gentreeg_CPPFLAGS =
-hamheuristic_CPPFLAGS =
-labelg_CPPFLAGS =
-linegraphg_CPPFLAGS =
-listg_CPPFLAGS =
-multig_CPPFLAGS =
-newedgeg_CPPFLAGS =
-NRswitchg_CPPFLAGS =
-pickg_CPPFLAGS =
-planarg_CPPFLAGS =
-ranlabg_CPPFLAGS =
-shortg_CPPFLAGS =
-showg_CPPFLAGS =
-subdivideg_CPPFLAGS =
-twohamg_CPPFLAGS =
-vcolg_CPPFLAGS =
-watercluster2_CPPFLAGS = -DMAXN=32
-blisstog_CPPFLAGS =      $(ZLIB_CFLAGS)
-checks6_CPPFLAGS =
-sumlines_CPPFLAGS =      $(GMP_CFLAGS)
+nauty_addedgeg_CPPFLAGS =
+nauty_amtog_CPPFLAGS =
+nauty_biplabg_CPPFLAGS =
+nauty_catg_CPPFLAGS =
+nauty_complg_CPPFLAGS =
+nauty_converseg_CPPFLAGS =
+nauty_copyg_CPPFLAGS =
+nauty_countg_CPPFLAGS =
+nauty_cubhamg_CPPFLAGS =
+nauty_deledgeg_CPPFLAGS =
+nauty_delptg_CPPFLAGS =
+nauty_directg_CPPFLAGS =
+nauty_dretodot_CPPFLAGS =
+nauty_dretog_CPPFLAGS =
+nauty_genbg_CPPFLAGS =         $(nautyW1_flavour_CPPFLAGS)
+nauty_genbgL_CPPFLAGS =        $(nautyL1_flavour_CPPFLAGS) -DMAXN1=30
+nauty_geng_CPPFLAGS =          $(nautyW1_flavour_CPPFLAGS)
+nauty_genquarticg_CPPFLAGS =   $(nautyL1_flavour_CPPFLAGS)
+nauty_genrang_CPPFLAGS =
+nauty_genspecialg_CPPFLAGS =
+nauty_gentourng_CPPFLAGS =     -DMAXN=24
+nauty_gentreeg_CPPFLAGS =
+nauty_hamheuristic_CPPFLAGS =
+nauty_labelg_CPPFLAGS =
+nauty_linegraphg_CPPFLAGS =
+nauty_listg_CPPFLAGS =
+nauty_multig_CPPFLAGS =
+nauty_newedgeg_CPPFLAGS =
+nauty_NRswitchg_CPPFLAGS =
+nauty_pickg_CPPFLAGS =
+nauty_planarg_CPPFLAGS =
+nauty_ranlabg_CPPFLAGS =
+nauty_shortg_CPPFLAGS =
+nauty_showg_CPPFLAGS =
+nauty_subdivideg_CPPFLAGS =
+nauty_twohamg_CPPFLAGS =
+nauty_vcolg_CPPFLAGS =
+nauty_watercluster2_CPPFLAGS = -DMAXN=32
+nauty_blisstog_CPPFLAGS =      $(ZLIB_CFLAGS)
+nauty_checks6_CPPFLAGS =
+nauty_sumlines_CPPFLAGS =      $(GMP_CFLAGS)
 
 dreadnaut_SOURCES = dreadnaut.c
-addedgeg_SOURCES = addedgeg.c
-amtog_SOURCES = amtog.c
-biplabg_SOURCES = biplabg.c
-catg_SOURCES = catg.c
-complg_SOURCES = complg.c
-converseg_SOURCES = converseg.c
-copyg_SOURCES = copyg.c
-countg_SOURCES = testg.c
-cubhamg_SOURCES = cubhamg.c
-deledgeg_SOURCES = deledgeg.c
-delptg_SOURCES = delptg.c
-directg_SOURCES = directg.c
-dretodot_SOURCES = dretodot.c
-dretog_SOURCES = dretog.c
-genbg_SOURCES = genbg.c
-genbgL_SOURCES = genbg.c
-geng_SOURCES = geng.c
-genquarticg_SOURCES = genquarticg.c
-genrang_SOURCES = genrang.c
-genspecialg_SOURCES = genspecialg.c
-gentourng_SOURCES = gentourng.c
-gentreeg_SOURCES = gentreeg.c
-hamheuristic_SOURCES = hamheuristic.c
-labelg_SOURCES = labelg.c
-linegraphg_SOURCES = linegraphg.c
-listg_SOURCES = listg.c
-multig_SOURCES = multig.c
-newedgeg_SOURCES = newedgeg.c
-NRswitchg_SOURCES = NRswitchg.c
-pickg_SOURCES = testg.c
-planarg_SOURCES = planarg.c planarity.c
-ranlabg_SOURCES = ranlabg.c
-shortg_SOURCES = shortg.c
-showg_SOURCES = showg.c
-subdivideg_SOURCES = subdivideg.c
-twohamg_SOURCES = twohamg.c
-vcolg_SOURCES = vcolg.c
-watercluster2_SOURCES = watercluster2.c
-blisstog_SOURCES = blisstog.c
-checks6_SOURCES = checks6.c
-sumlines_SOURCES = sumlines.c
+nauty_addedgeg_SOURCES = addedgeg.c
+nauty_amtog_SOURCES = amtog.c
+nauty_biplabg_SOURCES = biplabg.c
+nauty_catg_SOURCES = catg.c
+nauty_complg_SOURCES = complg.c
+nauty_converseg_SOURCES = converseg.c
+nauty_copyg_SOURCES = copyg.c
+nauty_countg_SOURCES = testg.c
+nauty_cubhamg_SOURCES = cubhamg.c
+nauty_deledgeg_SOURCES = deledgeg.c
+nauty_delptg_SOURCES = delptg.c
+nauty_directg_SOURCES = directg.c
+nauty_dretodot_SOURCES = dretodot.c
+nauty_dretog_SOURCES = dretog.c
+nauty_genbg_SOURCES = genbg.c
+nauty_genbgL_SOURCES = genbg.c
+nauty_geng_SOURCES = geng.c
+nauty_genquarticg_SOURCES = genquarticg.c
+nauty_genrang_SOURCES = genrang.c
+nauty_genspecialg_SOURCES = genspecialg.c
+nauty_gentourng_SOURCES = gentourng.c
+nauty_gentreeg_SOURCES = gentreeg.c
+nauty_hamheuristic_SOURCES = hamheuristic.c
+nauty_labelg_SOURCES = labelg.c
+nauty_linegraphg_SOURCES = linegraphg.c
+nauty_listg_SOURCES = listg.c
+nauty_multig_SOURCES = multig.c
+nauty_newedgeg_SOURCES = newedgeg.c
+nauty_NRswitchg_SOURCES = NRswitchg.c
+nauty_pickg_SOURCES = testg.c
+nauty_planarg_SOURCES = planarg.c planarity.c
+nauty_ranlabg_SOURCES = ranlabg.c
+nauty_shortg_SOURCES = shortg.c
+nauty_showg_SOURCES = showg.c
+nauty_subdivideg_SOURCES = subdivideg.c
+nauty_twohamg_SOURCES = twohamg.c
+nauty_vcolg_SOURCES = vcolg.c
+nauty_watercluster2_SOURCES = watercluster2.c
+nauty_blisstog_SOURCES = blisstog.c
+nauty_checks6_SOURCES = checks6.c
+nauty_sumlines_SOURCES = sumlines.c
 
 dreadnaut_LDADD =     $(NAUTY_LDADD)
-addedgeg_LDADD =      $(NAUTY_LDADD)
-amtog_LDADD =         $(NAUTY_LDADD)
-biplabg_LDADD =       $(NAUTY_LDADD)
-catg_LDADD =          $(NAUTY_LDADD)
-complg_LDADD =        $(NAUTY_LDADD)
-converseg_LDADD =     $(NAUTY_LDADD)
-copyg_LDADD =         $(NAUTY_LDADD)
-countg_LDADD =        $(NAUTY_LDADD)
-cubhamg_LDADD =       $(NAUTY_LDADD)
-deledgeg_LDADD =      $(NAUTY_LDADD)
-delptg_LDADD =        $(NAUTY_LDADD)
-directg_LDADD =       $(NAUTY_LDADD)
-dretodot_LDADD =      $(NAUTY_LDADD) $(LIBM)
-dretog_LDADD =        $(NAUTY_LDADD)
-genbg_LDADD =         $(NAUTYW1_LDADD)
-genbgL_LDADD =        $(NAUTYL1_LDADD)
-geng_LDADD =          $(NAUTYW1_LDADD)
-genquarticg_LDADD =   $(NAUTYL1_LDADD)
-genrang_LDADD =       $(NAUTY_LDADD)
-genspecialg_LDADD =   $(NAUTY_LDADD)
-gentourng_LDADD =     $(NAUTY_LDADD)
-gentreeg_LDADD =      $(NAUTY_LDADD)
-hamheuristic_LDADD =  $(NAUTY_LDADD)
-labelg_LDADD =        $(NAUTY_LDADD)
-linegraphg_LDADD =    $(NAUTY_LDADD)
-listg_LDADD =         $(NAUTY_LDADD)
-multig_LDADD =        $(NAUTY_LDADD)
-newedgeg_LDADD =      $(NAUTY_LDADD)
-NRswitchg_LDADD =     $(NAUTY_LDADD)
-pickg_LDADD =         $(NAUTY_LDADD)
-planarg_LDADD =       $(NAUTY_LDADD)
-ranlabg_LDADD =       $(NAUTY_LDADD)
-shortg_LDADD =        $(NAUTY_LDADD)
-showg_LDADD =
-subdivideg_LDADD =    $(NAUTY_LDADD)
-twohamg_LDADD =       $(NAUTY_LDADD)
-vcolg_LDADD =         $(NAUTY_LDADD)
-watercluster2_LDADD = $(NAUTY_LDADD)
-blisstog_LDADD =      $(NAUTY_LDADD) $(ZLIB_LIBS)
-checks6_LDADD =       $(NAUTY_LDADD)
-sumlines_LDADD =      $(GMP_LIBS)
+nauty_addedgeg_LDADD =      $(NAUTY_LDADD)
+nauty_amtog_LDADD =         $(NAUTY_LDADD)
+nauty_biplabg_LDADD =       $(NAUTY_LDADD)
+nauty_catg_LDADD =          $(NAUTY_LDADD)
+nauty_complg_LDADD =        $(NAUTY_LDADD)
+nauty_converseg_LDADD =     $(NAUTY_LDADD)
+nauty_copyg_LDADD =         $(NAUTY_LDADD)
+nauty_countg_LDADD =        $(NAUTY_LDADD)
+nauty_cubhamg_LDADD =       $(NAUTY_LDADD)
+nauty_deledgeg_LDADD =      $(NAUTY_LDADD)
+nauty_delptg_LDADD =        $(NAUTY_LDADD)
+nauty_directg_LDADD =       $(NAUTY_LDADD)
+nauty_dretodot_LDADD =      $(NAUTY_LDADD) $(LIBM)
+nauty_dretog_LDADD =        $(NAUTY_LDADD)
+nauty_genbg_LDADD =         $(NAUTYW1_LDADD)
+nauty_genbgL_LDADD =        $(NAUTYL1_LDADD)
+nauty_geng_LDADD =          $(NAUTYW1_LDADD)
+nauty_genquarticg_LDADD =   $(NAUTYL1_LDADD)
+nauty_genrang_LDADD =       $(NAUTY_LDADD)
+nauty_genspecialg_LDADD =   $(NAUTY_LDADD)
+nauty_gentourng_LDADD =     $(NAUTY_LDADD)
+nauty_gentreeg_LDADD =      $(NAUTY_LDADD)
+nauty_hamheuristic_LDADD =  $(NAUTY_LDADD)
+nauty_labelg_LDADD =        $(NAUTY_LDADD)
+nauty_linegraphg_LDADD =    $(NAUTY_LDADD)
+nauty_listg_LDADD =         $(NAUTY_LDADD)
+nauty_multig_LDADD =        $(NAUTY_LDADD)
+nauty_newedgeg_LDADD =      $(NAUTY_LDADD)
+nauty_NRswitchg_LDADD =     $(NAUTY_LDADD)
+nauty_pickg_LDADD =         $(NAUTY_LDADD)
+nauty_planarg_LDADD =       $(NAUTY_LDADD)
+nauty_ranlabg_LDADD =       $(NAUTY_LDADD)
+nauty_shortg_LDADD =        $(NAUTY_LDADD)
+nauty_showg_LDADD =
+nauty_subdivideg_LDADD =    $(NAUTY_LDADD)
+nauty_twohamg_LDADD =       $(NAUTY_LDADD)
+nauty_vcolg_LDADD =         $(NAUTY_LDADD)
+nauty_watercluster2_LDADD = $(NAUTY_LDADD)
+nauty_blisstog_LDADD =      $(NAUTY_LDADD) $(ZLIB_LIBS)
+nauty_checks6_LDADD =       $(NAUTY_LDADD)
+nauty_sumlines_LDADD =      $(GMP_LIBS)
 
 libnauty_la_LD_VERSION_SCRIPT =
 libnautyA1_la_MAP =
--- a/runalltests
+++ b/runalltests
@@ -30,7 +30,7 @@
 eval $cmd <"$in" 2>$out2 >$out1
 
 if [ -d $top_workingdir/.libs ]; then
-   sed -i 's@'$top_workingdir'/.libs/lt-@./@' $out2
+   sed -i 's@'$top_workingdir'/.libs/lt-nauty-@./@' $out2
 fi
 
 LC_COLLATE=C sort $out2 >>$out1
@@ -82,13 +82,13 @@
 runonetest ./dreadtest4K nautest2.dre nautest2a.ans
 runonetest "./dreadtest4K -o As" nautest2.dre nautest2b.ans
 
-runonetest "./geng -ud1D7t 11" "" nautesta.ans
-runonetest "./geng -cd1f 10 | ./labelg -q" "" nautestb.ans
-runonetest "./genrang -r3 114 100 | ./countg --nedDr -q" "" nautestc.ans
-runonetest "./gentourng -q -z 8 | ./pickg -T3" "" nautestd.ans
-runonetest "./gentreeg -q -s 18 | ./countg -z6 --o" "" nauteste.ans
-runonetest "./geng -c -q 8 8 | ./directg -q -o | ./converseg -q | ./labelg -q" "" nautestf.ans
-runonetest "./genbg -q 4 3 10 | ./vcolg -m6 -T" "" nautestg.ans
+runonetest "./nauty-geng -ud1D7t 11" "" nautesta.ans
+runonetest "./nauty-geng -cd1f 10 | ./nauty-labelg -q" "" nautestb.ans
+runonetest "./nauty-genrang -r3 114 100 | ./nauty-countg --nedDr -q" "" nautestc.ans
+runonetest "./nauty-gentourng -q -z 8 | ./nauty-pickg -T3" "" nautestd.ans
+runonetest "./nauty-gentreeg -q -s 18 | ./nauty-countg -z6 --o" "" nauteste.ans
+runonetest "./nauty-geng -c -q 8 8 | ./nauty-directg -q -o | ./nauty-converseg -q | ./nauty-labelg -q" "" nautestf.ans
+runonetest "./nauty-genbg -q 4 3 10 | ./nauty-vcolg -m6 -T" "" nautestg.ans
 
 echo " "
 
